content/posts/pull-from-upstream.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Automatic pulling of upstream releases to Fedora"
+date: 2023-01-23T09:23:44+01:00
+weight: 92
+---
+
+In the previous year, we [automated](/posts/downstream-automation) the Fedora downstream release process in Packit.
+The first step of the release process, propagating the upstream release to Fedora,
+is covered by the [`propose_downstream`](/docs/configuration/#propose_downstream)
+job, which works on top of the CLI command [`packit propose-downstream`](/docs/cli/propose-downstream).
+This job updates the sources in Fedora, the spec file, and other needed files and creates pull requests with the changes
+in the dist-git repository.
+
+The downside of this job is that for its execution, users need to install Packit Service GitHub/GitLab
+app since this job reacts only to GitHub/GitLab release webhooks.
+However, the person who maintains the package in Fedora may not be the upstream maintainer and may not have admin access
+to the upstream GitHub/GitLab repository.
+
+To cover this case, we came up with a new job called `pull_from_upstream`, which aims to update Fedora dist-git similarly
+to `propose_downstream`, but is configured directly in the dist-git repository.
+Let's now look at how to set it up and how it works.
+
+## Setup
+
+### Upstream release monitoring
+
+`pull_from_upstream` job reacts to a new bug in [Bugzilla](https://bugzilla.redhat.com/) about a new upstream version
+of a project. The bug is automatically created by
+[Upstream Release Monitoring](https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring/).
+To enable the Upstream Release Monitoring:
+{{< columns >}}
+
+1. Add the upstream project (if it is not there yet)
+   to [Anitya](https://release-monitoring.org/) and configure the mapping to a Fedora package:
+   ![Project in Anitya](/images/anitya-project.png)
+   <--->
+2. Enable the monitoring in the dist-git repository ([Fedora Package Sources](https://src.fedoraproject.org)):
```

Comment:
One small suggestion, could you edit into the screenshot an arrow pointing at the `Monitoring` option so that it's clear what the text is talking about?

Reply:
good point, will do :+1: